commit 89e2e4a2f24f54c14cf7b241daea375324a0a5eb
Author: Bob Peterson <rpeterso@redhat.com>
Date:   Wed Mar 6 10:29:38 2013 -0700

    fsck.gfs2: shorten some debug messages in lost+found
    
    This patch changes the debug output of lost+found such that it
    only prints the block number in hexadecimal. This shortens the output
    and makes debug output easier to read.
    
    rhbz#902920

diff --git a/gfs2/fsck/lost_n_found.c b/gfs2/fsck/lost_n_found.c
index 42d97ee..388787d 100644
--- a/gfs2/fsck/lost_n_found.c
+++ b/gfs2/fsck/lost_n_found.c
@@ -32,11 +32,9 @@ static void add_dotdot(struct gfs2_inode *ip)
 	if (di && valid_block(sdp, di->dotdot_parent.no_addr)) {
 		struct gfs2_inode *dip;
 
-		log_debug(_("Directory %lld (0x%llx) already had a "
-			    "\"..\" link to %lld (0x%llx).\n"),
+		log_debug(_("Directory (0x%llx) already had a "
+			    "\"..\" link to (0x%llx).\n"),
 			  (unsigned long long)ip->i_di.di_num.no_addr,
-			  (unsigned long long)ip->i_di.di_num.no_addr,
-			  (unsigned long long)di->dotdot_parent.no_addr,
 			  (unsigned long long)di->dotdot_parent.no_addr);
 		dip = fsck_load_inode(sdp, di->dotdot_parent.no_addr);
 		if (dip->i_di.di_num.no_formal_ino ==
@@ -74,15 +72,13 @@ static void add_dotdot(struct gfs2_inode *ip)
 	} else {
 		if (di)
 			log_debug(_("Couldn't find a valid \"..\" entry "
-				    "for orphan directory %lld (0x%llx): "
+				    "for orphan directory (0x%llx): "
 				    "'..' = 0x%llx\n"),
 				  (unsigned long long)ip->i_di.di_num.no_addr,
-				  (unsigned long long)ip->i_di.di_num.no_addr,
 				  (unsigned long long)di->dotdot_parent.no_addr);
 		else
-			log_debug(_("Couldn't find directory %lld (0x%llx) "
+			log_debug(_("Couldn't find directory (0x%llx) "
 				    "in directory tree.\n"),
-				  (unsigned long long)ip->i_di.di_num.no_addr,
 				  (unsigned long long)ip->i_di.di_num.no_addr);
 	}
 	if (gfs2_dirent_del(ip, "..", 2))
